hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenUpdateNftsSuite.java

```diff
@@ -76,26 +84,333 @@ final Stream<DynamicTest> idVariantsTreatedAsExpected() {
                         .signedBy(DEFAULT_PAYER, "multiKey")));
     }
 
+    /**
+     * <a href="https://github.com/hashgraph/hedera-improvement-proposal/blob/main/HIP/hip-850.md">HIP-850</a>
+     * Tests that the transaction fails if it is not signed by the metadata key or the supply key.
+     * @return the dynamic test
+     */
     @HapiTest
-    final Stream<DynamicTest> failsIfTokenHasNoMetadataKey() {
-        return defaultHapiSpec("failsIfTokenHasNoMetadataKey")
+    final Stream<DynamicTest> failsIfNoMetadataKeyOrSupplyKey() {
```

Comment:
nit : name can be `failsIfNoMetadataKeyOrSupplyKeySigns()`

Reply:
Updated test name.